arviz/plots/rankplot.py

```diff
@@ -0,0 +1,58 @@
+import numpy as np
+
+from ..data import convert_to_dataset
+from .plot_utils import (
+    _scale_fig_size,
+    xarray_var_iter,
+    default_grid,
+    _create_axes_grid,
+    make_label,
+)
+from ..utils import _var_names
+
+
+def plot_rank(data, var_names=None, coords=None, bins="auto", figsize=None, axes=None):
+
+    posterior_data = convert_to_dataset(data, group="posterior")
+    if coords is not None:
+        posterior_data = posterior_data.sel(**coords)
+    var_names = _var_names(var_names, posterior_data)
+    plotters = list(xarray_var_iter(posterior_data, var_names=var_names, combined=True))
+
+    if axes is None:
+        rows, cols = default_grid(len(plotters))
+
+        figsize, ax_labelsize, titlesize, _, _, _ = _scale_fig_size(
+            figsize, None, rows=rows, cols=cols
+        )
+        _, axes = _create_axes_grid(len(plotters), rows, cols, figsize=figsize, squeeze=False)
+
+    for ax, (var_name, selection, var_data) in zip(axes.ravel(), plotters):
+        ranks = var_data.argsort(axis=None).reshape(*var_data.shape)
```

Comment:
This will not give ranks.

Adhoc way would do argsort twice which give ranks, but we actually want to use `scipy.stats.rankdata` that averages the rank for equal values (this is needed for int parameters)

Reply:
good catch, thank you